ballerina-integrator/github-issue-to-salesforce/.choreo/instructions.md

```
@@ -0,0 +1,82 @@
+
+<details>
+<summary>Additional Configurations</summary>
+
+1. `triggerLabels`
+   - List of GitHub issue labels that trigger Salesforce case creation
+   - Example: `triggerLabels = ["bug", "support", "help wanted"]`
+
+2. `githubRepositories`
+   - List of GitHub repository URLs to monitor
+   - Example: 
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)